code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -426,7 +426,7 @@
 
 /datum/spellbook_entry/item/battlemage
 	name = "Battlemage Armour"
-	desc = "An ensorceled suit of armour, protected by a powerful shield. The shield can completely negate sixteen attacks before being permanently depleted."
+	desc = "An ensorceled suit of armour, protected by a powerful shield. The shield can completely negate sixteen attacks before being permanently depleted. Also comes with a pair of sandals and enchanted (and, more importantly, insulated) gloves."

```

Comment:
bro the enchant makes them insulated why add the text in brackets smh

Reply:
because "enchanted" doesn't tell you anything about what the gloves actually DO

Reply:
yeah bro they are enchanted so they are insulated

Reply:
yeah but how are you supposed to know that if you aren't a codediver